GitTfs.Vs2010/GitTfs.Vs2010.csproj

```diff
@@ -146,6 +143,6 @@
   </Target>
   -->
   <PropertyGroup>
-    <PostBuildEvent>xcopy /y /I "$(TargetDir)*.dll" "$(SolutionDir)GitTfs\$(OutDir)$(ProjectName)" /EXCLUDE:$(SolutionDir)files_to_ignore_during_post_build_copy.txt</PostBuildEvent>
+    <PostBuildEvent>xcopy /y /I "$(TargetPath)" "$(SolutionDir)GitTfs\$(OutDir)$(ProjectName)\*"</PostBuildEvent>
```

Comment:
ahha, i missed that it changed from `$(TargetDir)/...` to `$(TargetPath)`. Also, it sounds reasonable that the VsXXXX dlls don't have any unique dependencies (other than the TFS dlls that we don't ship).

Thanks for the explanation.